src/async_fuse/test/integration_tests.rs

```diff
@@ -518,53 +518,78 @@ fn test_bind_mount(fuse_mount_dir: &Path) -> anyhow::Result<()> {
     Ok(())
 }
 
-#[test]
-fn run_test() -> anyhow::Result<()> {
-    _run_test(DEFAULT_MOUNT_DIR, false)
+#[tokio::test(flavor = "multi_thread")]
+async fn run_test() {
+    if let Err(e) = _run_test(DEFAULT_MOUNT_DIR, false).await {
+        panic!("run test failed for error {}", e);
+    }
 }
 
 #[ignore]
-#[test]
-fn run_s3_test() -> anyhow::Result<()> {
-    _run_test(S3_DEFAULT_MOUNT_DIR, true)
+#[tokio::test(flavor = "multi_thread")]
+async fn run_s3_test() {
+    if let Err(e) = _run_test(S3_DEFAULT_MOUNT_DIR, false).await {
+        panic!("run test failed for error {}", e);
+    }
 }
 
-fn _run_test(mount_dir_str: &str, is_s3: bool) -> anyhow::Result<()> {
+#[allow(clippy::unwrap_used)]
+async fn _run_test(mount_dir_str: &str, is_s3: bool) -> anyhow::Result<()> {
     let mount_dir = Path::new(mount_dir_str);
-    let th = test_util::setup(mount_dir, is_s3)?;
+    let th = test_util::setup(mount_dir, is_s3).await.unwrap();
     info!("begin integration test");
 
-    test_symlink_dir(mount_dir).context("test_symlink_dir() failed")?;
-    test_symlink_file(mount_dir).context("test_symlink_file() failed")?;
+    test_symlink_dir(mount_dir)
+        .context("test_symlink_dir() failed")
+        .unwrap();
```

Comment:
use `?` instead of unwrap() . also change other places

Reply:
done